packages/neuron-ui/src/components/Send/index.tsx

```diff
@@ -149,36 +149,15 @@ const Send = () => {
     })
   }, [outputs, network])
 
-  let balancePrompt = null
-  if (ConnectionStatus.Offline === connectionStatus) {
-    balancePrompt = (
-      <span className={styles.balancePrompt} style={{ color: 'red' }}>
-        {t('sync.sync-failed')}
-      </span>
-    )
-  } else if (SyncStatus.SyncNotStart === syncStatus) {
-    balancePrompt = (
-      <span className={styles.balancePrompt} style={{ color: 'red' }}>
-        {t('sync.sync-not-start')}
-      </span>
-    )
-  } else if (
-    [SyncStatus.Syncing, SyncStatus.SyncPending].includes(syncStatus) ||
-    ConnectionStatus.Connecting === connectionStatus
-  ) {
-    balancePrompt = <span className={styles.balancePrompt}>{t('sync.syncing-balance')}</span>
-  }
-
   return (
-    <form style={{ padding: '39px 0 0 0' }} onSubmit={disabled ? undefined : onSubmit} data-wallet-id={walletID}>
+    <form onSubmit={disabled ? undefined : onSubmit} data-wallet-id={walletID}>
```

Comment:
It might be better to encapsulate the `disabled` check within `onSubmit` callback function for mainly two reasons:
1. This section of onSubmit binding would be more readable. Normally I would try to avoid putting too much logic in the JSX.
2. Later on, it would be easier to handle the extended logics along with the current `disabled` within the callback function.

Reply:
Sure, I'll do it later